software/supabase.yml

```
@@ -0,0 +1,13 @@
+name: Supabase
+website_url: https://supabase.com
+description: Firebase alternative built on Postgres. Includes auth, realtime, file store, and a vector db.
+licenses:
+  - Apache-2.0
+platforms:
+  - Docker
+tags:
+  - Database Management
```

```suggestion
tags:
  - Software Development - FaaS & Serverless
```

We already list a similar solution: https://awesome-selfhosted.net/#nhost (_Firebase Alternative with GraphQL. Get a database and backend configured and ready in minutes._) which is tagged `Software Development - FaaS & Serverless`. I think `Database Management` is traditionally used for web frontends/graphical DB management tools in the context of the list.